index.html

```diff
@@ -220,6 +219,75 @@ <h3>Skills</h3>
 			</div>
 		</section>
 
-		<script src="main.js"></script>
-	</body>
-</html>
+        <section id="contact">
```

Comment:
- [ ] As mentioned by the previous reviewer, there is an overlapping issue between the skills sections and the contact section, kindly fix this issue 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/213875058-f920581e-a616-449a-b357-5cb9ef2c4479.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/213875078-9f5d33d6-647d-4708-b1c8-c0e21da9650c.png)